staging/src/k8s.io/apimachinery/pkg/runtime/serializer/cbor/internal/modes/custom.go

```diff
@@ -140,7 +140,7 @@ func (cache *checkers) getCheckerInternal(rt reflect.Type, parent *path) (c chec
 	var wg sync.WaitGroup
 	wg.Add(1)
 	defer wg.Done()
-	c = checker{
+	lazy := checker{
```

Comment:
Do you actually need the named return at all?

Reply:
The return value needs to have an identifier to allow the placeholder value to delegate to it, and this function must always return a usable value, so it made sense to have a named return.

Reply:
Ah sorry I misread the function based on the diff.  Named return seems fine.